ffi/capi/bindings/dart/Bidi.g.dart

```diff
@@ -26,91 +27,99 @@ final class Bidi implements ffi.Finalizable {
   }
 
   static final _finalizer = ffi.NativeFinalizer(ffi.Native.addressOf(_icu4x_Bidi_destroy_mv1));
-
   /// Creates a new [`Bidi`] from locale data using compiled data.
-  ///
+///
   /// See the [Rust documentation for `new`](https://docs.rs/icu/latest/icu/properties/bidi/struct.BidiClassAdapter.html#method.new) for more information.
+
   factory Bidi() {
     final result = _icu4x_Bidi_create_mv1();
-    return Bidi._fromFfi(result, []);
-  }
+        return Bidi._fromFfi(result, []);
```

Comment:
issue: this is not indented correctly

Reply:
argh, I thought I fixed this problem but I think it got overwritten by a different change.